src/annotation/AdulthoodValidator.java

```diff
@@ -0,0 +1,29 @@
+package annotation;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.time.Period;
+
+public class AdulthoodValidator {
+    public  String [] validate(Object object) {
+        String [] errorMessages = new String[5];
+
+        Class<?> aClass = object.getClass();
+        Field[] declaredFields = aClass.getDeclaredFields();
+        for(Field field: declaredFields){
+            if(field.isAnnotationPresent(Adulthood.class)){
+                field.setAccessible(true);
+                Adulthood annotation = field.getAnnotation(Adulthood.class);
+                Customer customer = (Customer) object;
+                int fieldValue = customer.getAge();
```

Comment:
You don't have to get field value directly. You have to get the value of the field which is annotated in Adulthood annotation.